### DISCUSSION TITLE
Is this project not maintained anymore?

### DISCUSSION THREAD

**Discussion by ebisbe** [serverless-operations/serverless-step-functions#587] [General]
Just as the title says. I've just started using Step functions and I think it's the only option for the serverless framework. There's a quite a lot of issues and unresolved PR.

Pinging @horike37 @theburningmonk @zirkelc ( as they are the ones with more contributions to this project )

**Comment by zirkelc** [comment]
I'm not an official maintainer, but I'd like to help out in case this repo needs new maintainers.

**Comment by horike37** [comment]
@ebisbe @zirkelc 
Thank you for pinging me. No, not really but the pace for maintaining this plugin is decreasing actually. Therefore it would be great if we've got any active new maintainers. good ideas to solve this is always welcome.

I will take a look at the unresolved PRs tomorrow.

**Reply by ebisbe** [reply]
my current situation is that I've started using step functions and this plugin is the only one for serverless framework. I don't plan to migrate to cdk in a near future. So I can try to solve some issues ( fixing / enhancing code already ) and help with managing issues.  

Right now for me almost everything is working as expected. The only real issue is the amount of missing permissions I have. I guess that's related to allowing step functions to integrate to 200+ services.

**Reply by jappurohit041** [reply]
Hii @horike37,

If we are looking for any new active maintainers, than I would be highly interested to be a one of the maintainer for this plugin.

Yes @ebisbe, I agree, this plugin is very critical and continuous enhancement is required along with maintenance.

**Comment by louis0802** [comment]
Hi @horike37 @theburningmonk @zirkelc @jappurohit041, would you be able to help me review my [PR](https://github.com/serverless-operations/serverless-step-functions/pull/594) ? Thanks
